src/Entity/Probability.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Acquia\LiftClient\Entity;
+
+use Acquia\LiftClient\Exception\LiftSdkException;
+
+class Probability extends Entity
+{
+    /**
+     * @param array $array
+     */
+    public function __construct(array $array = [])
+    {
+        parent::__construct($array);
+    }
+
+    /**
+     * Sets the 'id' parameter.
+     *
+     * @param string $contentId
+     *
+     * @throws \Acquia\LiftClient\Exception\LiftSdkException
+     *
+     * @return \Acquia\LiftClient\Entity\Probability
+     */
+    public function setContentId($contentId)
+    {
+        if (!is_string($contentId)) {
+            throw new LiftSdkException('Argument must be an instance of string.');
+        }
+        $this['id'] = $contentId;
```

Comment:
Fixed, and this made me realize we need unit tests for what the SDK generates as a payload. Right now I'm only testing responses and the object generation to the Add method. In the add method it should also be able to check the request payload and verify that.